es-app/src/guis/GuiMenu.cpp

```diff
@@ -17,8 +17,6 @@
 #include "components/OptionListComponent.h"
 #include "components/MenuComponent.h"
 #include "VolumeControl.h"
-#include "scrapers/GamesDBScraper.h"
-#include "scrapers/TheArchiveScraper.h"
```

Comment:
FYI Removing both mentions here since these were from back when the list of scrapers was set directly here instead of calling getScraperList

https://github.com/Aloshi/EmulationStation/blob/5039b38d8dc04042463ef584fd2428ebda5b36c5/src/guis/GuiMenu.cpp#L49-L50